go/cmd/server/server_test.go

```diff
@@ -58,21 +67,67 @@ func getClient(ctx context.Context) (serving.ServingServiceClient, func()) {
 }
 
 func TestGetFeastServingInfo(t *testing.T) {
-	t.Skip("@todo(achals): feature_repo isn't checked in yet")
 	ctx := context.Background()
-	client, closer := getClient(ctx)
+	// Pregenerated using `feast init`.
+	dir := "../../internal/test"
```

Comment:
I think this is pretty fragile.. having the feature repo created in a separate command from the `go test` command. Can we have a `Setup` method create the feature repo here so that everything is self-contained and it gets executed as part of this test?

Reply:
Done, added methods for setup and cleanup